backend/corpora/common/utils/s3_utils.py

```diff
@@ -1,28 +1,46 @@
-import os
+import logging
+import typing
 
 import boto3
 from botocore.exceptions import ClientError
 
+logger = logging.getLogger(__name__)
 
-def generate_file_url(file_prefix: str, expiration: int = 3600, s3=None) -> str:
+
+def generate_file_url(bucket_name: str, file_prefix: str, expiration: int = 3600) -> typing.Union[str, None]:
```

Comment:
Can we make the default expiration 604800? That should the be AWS max.

Reply:
Guess it would really need to go in dataset_asset.py

Reply:
Done